program/src/main/System.vala

```diff
@@ -738,7 +752,9 @@ public class System
         }
 
         // load the new question from the file.
-        QuestionSet nextQuestion;
+        QuestionSet nextQuestion; //note 1- update progress bar here
+        progressBar.set_fraction(9/10); //PROGRESS_BAR_TEST
```

Comment:
Do you need to do something based on this comment? And another test

Reply:
I think that comment was made before we changed the progressbar design to be simply based on keypress updates. Removed!